auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/rest/AuthenticatorAPI.java

```diff
@@ -103,6 +103,7 @@ public String bdreLogin(HttpSession session,
 
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)